src/shared/export-json-file.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Provided an object and a filename, allows to generate a downloadable file
+ * containing a JSON-serialized version of that object
+ *
+ * @param _document - Test seam
+ * @return The contents of the downloaded file
+ */
+export function exportJSONFile(
```

Comment:
I'm wondering if `downloadJSONFile` might be more descriptive?

Reply:
It definitely is.